x/clp/keeper/executors.go

```diff
@@ -183,6 +192,8 @@ func (k Keeper) RemoveLiquidity(ctx sdk.Context, pool types.Pool, externalAssetC
 		}
 		err = k.bankKeeper.SendCoinsFromModuleToAccount(ctx, types.ModuleName, lpAddr, sendCoins)
 		if err != nil {
+			// I think you can trigger this if the lpAddr is a Blocked Address, see https://docs.cosmos.network/master/modules/bank/02_keepers.html
+			// (testing) this part I don't know how to trigger. because we verify if have coins in lp and all others verification already been handled.
```

Comment:
I would clean these comments up.  It can be triggered, but it is a pain, and probably not worth it.